tests/usgs_test.py

```diff
@@ -183,7 +187,26 @@ def test_normalize_empty_stations_df():
 
 def test_normalize_empty_data_df():
     df = pd.DataFrame()
-    df2 = usgs.normalize_usgs_station_data(df, truncate_seconds=False)
+    df2 = usgs.normalize_usgs_station_data(df)
 
     assert df2.empty
     assert isinstance(df2, pd.DataFrame)
+
+
+def test_request_nonexistant_data():
+    sta = usgs.get_usgs_stations()
+    sta = sta[
+        sta.site_no.isin(
+            [
+                "373707086300703",
+                "373707086300801",
+                "373707086300802",
+            ]
+        )
+    ]
+    ds = usgs.get_usgs_data(
+        usgs_metadata=sta,
+        endtime="2015-07-04",
+        period=1,
+    )
+    assert len(ds) == 0
```

Comment:
As before, `len(ds)` does not give info about the existence of data per se, but about the existence of data_vars. If you chnage the implementation in line 294-295, then this should be updated, too.